src/Core/IEPlugin/CARPMesh_Plugin.cc

```diff
@@ -244,8 +243,9 @@ FieldHandle SCIRun::CARPMesh_reader(LoggerHandle pr, const char *filename)
                    }
             }
 
+
             if (elem_type != "Tt") {
-              if (pr) pr->error("Mesh types other than Tet not supported");
+              if (pr) pr->error("Mesh types other than Tet and surface Tet (Tr) not supported");
```

Comment:
what is a surface tet? a tri?

Reply:
Accidentally left that in there from making the surface reader. I just pushed a fix.